sync3c.go

```diff
@@ -84,7 +87,11 @@ func main() {
 	extensionForMimeTypes["audio/opus"] = "opus"
 	extensionForMimeTypes["audio/mpeg"] = "mp3"
 
-	ci, err := findConferences("https://api.media.ccc.de/public/conferences")
+	if len(source) == 0 {
+		source = "media.ccc.de"
+	}
+	source := fmt.Sprintf("https://api.%s/public/conferences", source)
```

Comment:
While shadowing an existing variable `source` isn't really harmful, I'd still like to avoid in general. Let's just pass the result of this fmt.Sprintf directly to `findConferences` below.

Reply:
Sure, sorry, I forgot to mention but it was my first attempt at patching Go code :-)